_rules/audio-transcript-2eb176.md

```diff
@@ -5,6 +5,12 @@ rule_type: atomic
 description: |
   This rule checks that `audio` elements have a transcript that includes all auditory information.
 accessibility_requirements:
+  wcag20:1.2.1: # Audio-only and Video-only (Prerecorded) (A)
+    forConformance: true
+    secondary: true
```

Comment:
Should we start putting a reason here, as done in #2060 ?

Reply:
I will assign these questions to @WilcoFiers as not sure on the secondary requirements needs.

Reply:
I'll handle it in my PR. Don't worry about it here.